### DISCUSSION TITLE
No Grafana login after teslamate update

### DISCUSSION THREAD

**Discussion by CptCheesyCrust** [teslamate-org/teslamate#2489] [Q&A] [answered]
Hi,

i update teslamate and grafana via `docker-compose pull` to the latest version (on a RaspberryPi. Update to bullseye is done). After that, when i go for excample to "Dashboards > Drives" i only get to the Grafana login screen. There, teslamate:secret dosen't work.

I'm quite a noob at all RaspberryPi things , as you may have noticed. :-D

**Comment by cwanja** [comment, marked as answer]
Try `admin:admin`.

**Reply by cwanja** [reply]
You can also reset the password following the steps https://github.com/adriankumpf/teslamate/issues/2329#issuecomment-1014096467 in this issue #2329
